File: csw-config/csw-config-api/src/main/scala/csw/config/api/javadsl/JFileType.scala
package csw.config.api.javadsl

import csw.config.models.FileType

/**
 * Helper class for Java to get the handle of file types
 */
object JFileType {

  /**
   * Represents a file to be stored in annex store
   */
  val Annex: FileType = FileType.Annex

  /**
   * Represents a file to be stored in the repository normally
   */
  val Normal: FileType = FileType.Normal
}

File: csw-config/csw-config-api/src/main/java/csw/config/api/javadsl/JFileType.java
package csw.config.api.javadsl;

import csw.config.models.FileType;

/**
 * Helper class for Java to get the handle of file types
 */
public interface JFileType {

    /**
     * Represents a file to be stored in annex store
     */
    FileType Annex = FileType.Annex$.MODULE$;

    /**
     * Represents a file to be stored in the repository normally
     */
    FileType Normal = FileType.Normal$.MODULE$;
}
